recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/blktap.fc

```diff
@@ -25,3 +25,4 @@
 /var/run/blktap-control(/.*)?		gen_context(system_u:object_r:tapdisk_var_run_t,s0)
 /dev/xen/control		-c	gen_context(system_u:object_r:xen_device_t,s0)
 /dev/blktap-control		-c	gen_context(system_u:object_r:blktap_device_t,s0)
+/run/tapback.pid		--	gen_context(system_u:object_r:tapback_var_run_t,s0)
```

Comment:
Maybe use `/run/tapback\..*` here, since tapback seems to create more pidfiles that just tapback.pid.
Btw, `.` has to be escaped.

Reply:
Indeed.